content/python/concepts/math-module/math-sqrt.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: "math.sqrt()"
+Subjects:
+  - "Computer Science"
+  - "Data Science"
+Tags:
+  - "Functions"
+  - "Arithmetic"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-python-3"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+---
+
+## Definition
+
+Returns the square root of numeric input x.
+
+## Syntax
+
+```py
+math.sqrt(x)
+```
```

Comment:
It might be nice to note that the sqrt of a negative numeric input will return `nan`